assets/data/host.js

```diff
@@ -0,0 +1,76 @@
+export const fullActivityDescription = `
```

Comment:
I would like to understand the reasoning for the creation of this file, to begin with, actually.  This is all static data that ultimately gets dumped back into the HTML.  Bringing it over into a dynamic format (JS) and then treating it statically doesn't seem productive to me, and makes it drastically easier for ill-guided maintainers to resort to a large class of hacky solutions and slip in (improbable) programming errors.  Basically, it makes it easier to do the wrong thing, and I'm not clear on what good things it enables.

I would be interested in understanding why you moved this out of the HTML.

Reply:
File has been deleted - we talked about this together last Friday, I believe.